shared/locales/intl/unitformat.en-US.properties

```diff
@@ -37,3 +37,43 @@ duration-month-narrow[two]    = {{ value }}mths
 duration-month-narrow[few]    = {{ value }}mths
 duration-month-narrow[many]   = {{ value }}mths
 duration-month-narrow[other]  = {{ value }}mths
+
+digital-byte-short            = {[ plural(value) ]}
```

Comment:
I'm slightly confused by this naming.  In particular, by the use of the `groupName` prefix, like `duration-`.  The headers from http://www.unicode.org/cldr/charts/27/summary/pl.html#5556 seem to be unique even without the group names ('pages' as CLDR calls them).

I don't mind putting them in the string id, just want to better understand the strategy for extending this.  For instance, https://github.com/unicode-cldr/cldr-units-full/blob/master/main/en/units.json uses `energy-joule` when the CLDR raw data says [Energy and Power, energy-joule], but then it uses `mass-ton` for [Mass and Weight,	ton].

Then again, this is perhaps not important as these translations are temporary—in the future they will come directly from Intl.